manifests/c/Coursier/Coursier/2.1.4/Coursier.Coursier.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Coursier.Coursier
+PackageVersion: 2.1.4
+Installers:
+- InstallerUrl: https://github.com/coursier/coursier/releases/download/v2.1.4/cs-x86_64-pc-win32.zip
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cs-x86_64-pc-win32.exe
+    PortableCommandAlias: cs
+  InstallerSha256: 13525D0D4636818FC366E3647C857260A868DEF8FF7E944907ED0DE983E54610
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64

```

```suggestion
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  - PackageIdentifier: Oracle.JavaRuntimeEnvironment
```
Hi @superusercode,

The error suggests that we should have a `JRE` as a dependency, but there are many implementation. We have plans for a way to specify multiple or any of a class of dependency, but for now we can omit this if you want.